src/plugins/graph/models/data-configuration-model.ts

```diff
@@ -108,12 +108,16 @@ export const DataConfigurationModel = types
      * The rightNumeric attribute description is also not returned.
      */
     get attributeDescriptions() {
-      const descriptions = {...getSnapshot(self._attributeDescriptions)};
-      delete descriptions.rightNumeric;
-      if (self._yAttributeDescriptions.length > 0) {
-        descriptions.y = self._yAttributeDescriptions[0];
+      if (isAlive(self)) {
```

Comment:
In the CODAP code, we try _not_ to use `isAlive()` in models in favor of identifying the upstream cause of accessing the defunct model, e.g. something like use of `mstReaction()` instead of `reaction()`.

Reply:
I'd agree "in models" it is best to figure out the cause and fix it. In components I think there are some cases where isAlive is the only option. In particular within the render method of observable components. The render method is basically reaction but there isn't anyway to dispose of it other than removing the component from the react component tree. However there are cases where that component removal happens too late so in those cases I think isAlive is the only option.

Reply:
I backed out this change, but I am not sure how to figure out what the upstream is. 
The backtrace has this `data-configuration-model` call, a bunch of mobx methods, and then `treeManager.goToHistoryEntry`.  It appears that there is a reaction in there (there's a stack frame for `runReaction_`) but no hint what reaction it is.

Reply:
Generally speaking, the best approach is to find the autorun or reaction that's causing the problem. In CODAP we've adopted the policy/convention that all MobX reactions should provide a `name` in the `options`. This `name` is visible in the stack trace. So first, check those reaction-related functions in the stack trace to see if you can find a `name`. If not, you can add the `name` option to any autoruns or reactions in the neighborhood that don't already have them and try again. You can also add `isAlive()` tests inside the reactions (for debugging purposes).